packages/core-p2p/src/network-state.ts

```diff
@@ -53,7 +41,7 @@ class QuorumDetails {
     /**
      * Number of peers with a different last block id.
      */
-    public peersDifferentBlockId = 0;
+    public peersForked = 0;
```

Comment:
The comment "Number of peers with a different last block id" needs an update. Something among the lines of "number of peers that are on a different chain (forked)"